capplet/gsm-properties-dialog.c

```diff
@@ -808,3 +815,47 @@ gsm_properties_dialog_new (void)
 
         return GTK_WIDGET (object);
 }
+
+static gboolean
+gsm_properties_dialog_page_scroll_event_cb (GtkWidget        *widget,
+                                            GdkEventScroll   *event)
+
+{
+        GtkNotebook *notebook = GTK_NOTEBOOK(widget);
+        GtkWidget *child, *event_widget, *action_widget;
+
+        child = gtk_notebook_get_nth_page(notebook, gtk_notebook_get_current_page(notebook));
```

Comment:
missing a space between the function names and the opening parenthesis, and so on.

Reply:
Done too. Need to change the others in same case because use of copy/paste...